source/_docs/sso-organizations.md

```diff
@@ -59,3 +59,6 @@ Open a support ticket with Pantheon with the following:
 ## Troubleshooting
 ### Cannot Authenticate with Username/Password When Creating a New Machine Token
 If you are a member of a SAML-enabled organization, and the password field does not disappear after you enter your username and password, you'll need to log out of your active session, log back in, and try again. This can occur if you have two accounts with different email addresses and have not logged out of an active session.
+
+### Cannot Login Using Google Account
+Members of organizations that use single sign-on are unable to login to Pantheon with their Google account, and must log in using their company email address.
```

Comment:
Social Login is disabled for users in organizations using single sign-on. SAML allows organizations to centrally manage authentication policies, so allowing Social Login could circumvent that policy.